source/includes/api/tables/progress-response.rst

```diff
@@ -128,3 +128,71 @@
 
        - When ``mongosync`` starts up, this field returns ``null`` until
          ``mongosync`` identifies the coordinator.
+
+   * - ``verification``
+     - document
+     - Describes the verification process on the source and destination clusters.
+
+       .. versionadded:: 1.8
+
+   * - ``verification.source``
+     - document
+     - Describes the verification process on the source cluster.
+
+       .. versionadded:: 1.8
+
+   * - ``verification.source`` ``.lagTimeSeconds``
```

Comment:
Remove extra space here? Looks odd as is.

Reply:
@jason-price-mongodb, We need a space or newline character there as a workaround for a parser/renderer issue.  It's only used in tables where the text of an inline code element runs overlong.